src/types/teams.js

```diff
@@ -24,11 +22,11 @@ export type Team = {|
     name: string,
     description: string,
     email: string,
-    type: TeamType,
     company_name: string,
     allowed_domains: string,
     invite_id: string,
     allow_open_invite: boolean,
```

Comment:
I keep allow_open_invite in the type definition because backward compatibility with older server versions. This is used in one place which is not typed for now, but will be in the future and will need the field.